src/orders/components/OrderRefundUnfulfilledProducts/OrderRefundUnfulfilledProducts.tsx

```diff
@@ -0,0 +1,237 @@
+            line => {
+              const selectedLineQuantity = data.refundedProductQuantities.find(
+                refundedLine => refundedLine.id === line.id
+              );
+              const lineQuantity = line?.quantity - line?.quantityFulfilled;
+              const isError =
+                Number(selectedLineQuantity?.value) > lineQuantity ||
+                Number(selectedLineQuantity?.value) < 0;
+
+              return (
+                <TableRow key={line?.id}>
+                  <TableCellAvatar thumbnail={line?.thumbnail?.url}>
+                    {line?.productName ? line?.productName : <Skeleton />}
+                  </TableCellAvatar>
+                  <TableCell>
+                    {line?.unitPrice ? (
+                      <Money money={line?.unitPrice.gross} />
+                    ) : (
+                      <Skeleton />
+                    )}
+                  </TableCell>
+                  <TableCell className={classes.colQuantity}>
+                    {lineQuantity ? (
+                      <TextField
+                        disabled={disabled}
+                        type="number"
+                        inputProps={{
+                          className: classes.quantityInnerInput,
+                          "data-test": "quantityInput",
+                          "data-test-id": line?.id,
+                          max: lineQuantity.toString(),
+                          min: 0,
+                          style: { textAlign: "right" }
+                        }}
+                        fullWidth
+                        value={selectedLineQuantity?.value}
+                        onChange={event =>
+                          onRefundedProductQuantityChange(
+                            line.id,
+                            event.target.value
+                          )
+                        }
+                        InputProps={{
+                          endAdornment: lineQuantity && (
+                            <div className={classes.remainingQuantity}>
+                              / {lineQuantity}
+                            </div>
+                          )
+                        }}
+                        error={isError}
+                        helperText={
+                          isError &&
+                          intl.formatMessage({
+                            defaultMessage: "Improper value",
+                            description: "error message"
+                          })
+                        }
+                      />
+                    ) : (
+                      <Skeleton />
+                    )}
+                  </TableCell>
+                  <TableCell>
+                    {(lineQuantity && line?.unitPrice.gross && (
+                      <Money
+                        money={{
+                          ...line.unitPrice.gross,
+                          amount:
+                            (line.unitPrice.gross.amount || 0) *
```

Comment:
Use nullish coalescing operator (`??`) when `0` is a valid value in the first place